carapax-i18n/README.md

```diff
@@ -0,0 +1,28 @@
+# carapax-i18n
+
+i18n handler for [carapax](../carapax)
```

Comment:
Change this to full URL, it won't work on crates.io otherwise

Reply:
There are relative paths in other libraries, too